bindata/network/multus/multus.yaml

```diff
@@ -0,0 +1,78 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: multus-cni-config
+  namespace: multus
+  labels:
+    app: multus
+data:
+  70-multus.conf: |
+    {
+      "name": "multus",
+      "type": "multus",
+      "delegates": [{
+        "type": "{{.CNIPluginName}}",
```

Comment:
Can we use the Multus default CNI configuration directory? Or do we need to hard-code the name?